rotkehlchen/data/all_assets.meta

```diff
@@ -1 +1 @@
-{"md5": "7969237a9e90ff27d35c29e2ff2fe895", "version": 47}
```

Comment:
you are going to add a token upgrade anyway for other assets later. So don't bump the version yet. Do it after all token changes are done.